app/partials/errorDetail-Q029.html

```diff
@@ -11,11 +11,11 @@
             <td class="col-group-first" aria-hidden="true">&nbsp;</td>
         </tr>
         <tr>
-            <th scope="col" class="id">LAR Number</th>
-            <th scope="col" class="state_code">State Code</th>
-            <th scope="col" class="county_code">County Code</th>
-            <th scope="col" class="census_tract">Census Tract</th>
-            <th scope="col" class="recommended col-group-first">MSA/MD</th>
+            <th scope="col" role="columnheader" class="id" ng-class="{'cf-icon icon-up': sortedBy === 'properties[\'LAR number\']' && !reverse, 'cf-icon icon-down': sortedBy === 'properties[\'LAR number\']' && reverse}" ng-click="sort('properties[\'LAR number\']')" ng-attr-aria-sort="{{isSortedBy('properties[\'LAR number\']')}}">LAR Number</th>
```

Comment:
You don't need to use role="columnheader" here because we are already conveying the same information by using `<th scope="row"`. 
ref: http://www.w3.org/TR/wai-aria/roles#columnheader